shynet/shynet/settings.py

```diff
@@ -112,6 +115,29 @@
         }
     }
 
+# Solution to removal of Heroku DB Injection
+if 'DATABASE_URL' in os.environ:
+    if 'DATABASES' not in locals():
+        DATABASES = {}
+    url = urlparse.urlparse(os.environ['DATABASE_URL'])
+
+    # Ensure default database exists.
+    DATABASES['default'] = DATABASES.get('default', {})
+
+    # Update with environment configuration.
+    DATABASES['default'].update({
+        'NAME': url.path[1:],
+        'USER': url.username,
+        'PASSWORD': url.password,
+        'HOST': url.hostname,
+        'PORT': url.port,
+    })
+    if url.scheme == 'postgres':
+      DATABASES['default']['ENGINE'] = 'django.db.backends.postgresql_psycopg2'
+
+    if url.scheme == 'mysql':
+      DATABASES['default']['ENGINE'] = 'django.db.backends.mysql'
```

Comment:
I'm not comfortable with adding MySQL support. I have no real way to test Shynet on MySQL, and I can't really imagine any scenarios when someone would _need_ to use MySQL over Postgres (except in orgs with a lot of red tape, in which case Shynet probably isn't the best option anyway).

Furthermore, Shynet relies on some of Postgres' more advanced aggregation and annotation capabilities to calculate stats _inside_ the database that I'm not sure MySQL supports.

Reply:
Any chance we can remove the MySQL part completely, instead of just commenting it out?

Reply:
No problem 👍